docs/resources/lakeview_dashboard.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Workspace"
+---
+# databricks_lakeview_dashboard Resource
+
+This resource allows you to manage Databricks [Lakeview Dashboards](https://docs.databricks.com/api/workspace/lakeview). To manage [Lakeview Dashboards](https://docs.databricks.com/api/workspace/lakeview) you must have a warehouse access on your databricks workspace.
+
+## Example Usage
+
+Dashboard using `serialized_dashboard` attribute:
+
+```hcl
+resource "databricks_lakeview_dashboard" "dashboard" {
+			display_name			= 	"New Dashboard"
+			warehouse_id			=	"{env.TEST_DEFAULT_WAREHOUSE_ID}"
+			serialized_dashboard	        =	"{\"pages\":[{\"name\":\"new_name\",\"displayName\":\"New Page\"}]}"
```

Comment:
should just look up the sql warehouse from a data source


```suggestion
data "databricks_sql_warehouse" "starter" {
  name = "Starter Warehouse"
}

resource "databricks_lakeview_dashboard" "dashboard" {
  display_name			= 	"New Dashboard"
  warehouse_id			=	data.databricks_sql_warehouse.starter.id
  serialized_dashboard	        =	"{\"pages\":[{\"name\":\"new_name\",\"displayName\":\"New Page\"}]}"
```

Reply:
this page will be part of Terraform registry documentation, so we want a relatively self contained example. The approach of environment variable is specific to our acceptance tests, so won't be of use for other customers.

The Starter Warehouse comes by default with all workspaces, but I think we deleted that from our internal deco workspaces